LR_Tourist/BLL/Model/IEntityBaseDTO.cs

```diff
@@ -0,0 +1,8 @@
+﻿
+namespace BLL.Model
+{
+    public interface IEntityBaseDTO
```

Comment:
It looks strange because all your models here use `IEntityBase`. I guess it's a mistake.